src/components/LegacyFeeDrawer.tsx

```diff
@@ -0,0 +1,183 @@
+import BigNumber from 'bignumber.js'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { LayoutAnimation, StyleSheet, Text, View } from 'react-native'
+import CurrencyDisplay, { FormatType } from 'src/components/CurrencyDisplay'
+import Expandable from 'src/components/Expandable'
+import { EncryptionFeeIcon, ExchangeFeeIcon, SecurityFeeIcon } from 'src/components/FeeIcon'
+import LineItemRow from 'src/components/LineItemRow'
+import Touchable from 'src/components/Touchable'
+import { CurrencyInfo } from 'src/localCurrency/types'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Currency } from 'src/utils/currencies'
+
+interface Props {
+  isEstimate?: boolean
+  currency?: Currency
+  isExchange?: boolean
+  securityFee?: BigNumber
+  exchangeFee?: BigNumber
+  dekFee?: BigNumber
+  showDekfee?: boolean
+  feeLoading?: boolean
+  feeHasError?: boolean
+  totalFee?: BigNumber
+  testID?: string
+  currencyInfo?: CurrencyInfo
+  showLocalAmount?: boolean
+}
+
+/**
+ * @deprecated use FeeDrawer instead
+ */
+export default function LegacyFeeDrawer({
```

Comment:
I'm assuming that this file is completely unchanged, and just a rename? (which GH doesn't pick up since `FeeDrawer.tsx` still exists)

Reply:
yep